test/blueprints/Do.Test.Blueprints.Service/Singleton.cs

```diff
@@ -1,19 +1,30 @@
-﻿using Do.Database;
+﻿using Do.Communication;
+using Do.Database;
 
 namespace Do.Test;
 
 public class Singleton(
     TimeProvider _timeProvider,
     Func<Entity> _newEntity,
     ITransaction _transaction,
-    Func<OperationWithGenericParameter<Entity>> _newOperationWithGenericParameter
+    Func<OperationWithGenericParameter<Entity>> _newOperationWithGenericParameter,
+    IClient<Singleton> _client
 ) : SingletonBase(_timeProvider), IInterface
 {
     internal void TestOperationWithGenericParameter()
     {
         _newOperationWithGenericParameter().With().Execute();
     }
 
+    public async Task<dynamic> TestClient()

```

Comment:
let's return a statically typed class instance to make a better demonstration, e.g., create a record named `PullRequest` and fill that object using serialization instead of getting content as object